frontend/src/app/homepage-offer/offers.service.ts

```diff
@@ -10,28 +10,32 @@ import { environment } from '../../environments/environment';
 
 @Injectable()
 export class OffersService {
-  private url = `${environment.apiRoot}/offers/`;
+  private _url = `${environment.apiRoot}/offers/`;
   constructor (private http: Http) { }
 
   getOffers() {
-    return this.http.get(this.url, { withCredentials: true } )
+    return this.http.get(this._url, { withCredentials: true } )
       .map((res: Response) => res.json())
       .map(offers => {
         for (const offer of offers) {
-          this.loadDefaultImage(offer);
+          this._loadDefaultImage(offer);
         }
         return offers;
       });
   }
 
   getOffer(id: number): Observable<Offer> {
-    return this.http.get(`${this.url}${id}/`, { withCredentials: true })
+    return this.http.get(`${this._url}${id}/`, { withCredentials: true })
       .map((res: Response) => res.json())
-      .map(offer => this.loadDefaultImage(offer));
+      .map(offer => this._loadDefaultImage(offer));
   }
 
-  loadDefaultImage(offer: Offer): Offer {
-    if (offer.image === null) {
+  private _loadDefaultImage(offer: Offer): Offer {
+    if (isDevMode() && !offer.image) {
```

Comment:
I'm more a fan of:
```
if (!offer.image) {
  if (environments.randomOfferImages) {
    ...
 } else {
    ...
}
}
```

Reply:
Sounds reasonable, I will do that.

Reply:
Should I add randomOfferImages property to every environment file that we have?